server/index.js

```diff
@@ -122,6 +131,30 @@ app.get('/spotify/recs/:artistId/:trackId/:genre', (req, res) => {
     });
 });
 
+app.post('/spotify/create-playlist', ensureAuthenticated, (req, res, next) => {
+  spotifyApi.createPlaylist('Songify', { description: 'Your Songify recommendation playlist', public: false })
+    .then(function (data) {
+      // eslint-disable-next-line
+      console.log('Created playlist!');
+      res.json(data);
+    }, function (err) {
+      // eslint-disable-next-line
+      console.log('Something went wrong!', err);
+    });
+
+});
+app.post('/spotify/addTracks/:playlistId/:trackId', ensureAuthenticated, (req, res, next) => {
+  const { playlistId, trackId } = req.params;
+  spotifyApi.addTracksToPlaylist(playlistId, trackId.split(','))
+    .then(function (data) {
+      // eslint-disable-next-line
+      console.log('Added tracks to playlist!');
+    }, function (err) {
+      // eslint-disable-next-line
+      console.log('Something went wrong!', err);
+    });
```

Comment:
Please clean out all console logs and comments. If you receive an error, pass it to `next()`